src/mumble/AudioInput.cpp

```diff
@@ -384,6 +384,14 @@ void AudioInput::addMic(const void *data, unsigned int nsamp) {
 					if (qlEchoFrames.isEmpty()) {
 						iJitterSeq = 0;
 						iMinBuffered = 1000;
+					} else if (qlEchoFrames.count() >= 100) {
```

Comment:
Well, the code is weird.

iMinBuffered is, for every call to addMic(), updated to reflect the minimum buffer size we've encountered.

The drift correction code only runs for iJitterSeq > 100 (1 second must have passed) && iMinBuffered > 1 (more than 1 frame queued).

On my machine, iMinBuffered eventually reaches 1, and the drift correction is never run again. In this state, I have around ~1-6 frames in the queue, on every addMic() call. Sometimes, I even deplete the queue, in which case the isEmpty() state is entered, resetting things to iJitterSeq=0, iMinBuffered=1000. However, since we're already running at this point, the result is mostly that iMinBuffered is almost immediately set to a 'small' value (~1-6).

Once everything is 'settled' it mostly doesn't get run ever again.

I don't really know how to document, because I don't know what the intent of the code is...